<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCorrelation</class>
 <widget class="QDialog" name="AddCorrelation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new correlation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>38</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Reynolds</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>27</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fluid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>190</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Section type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Angle</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="exprBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>16</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="authorBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>44</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="reMinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="reMaxBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="fluidBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="angleBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="sectionBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>220</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Border type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="borderBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>48</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prandtl  </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>20</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="prMinBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <double>9999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="prMaxBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <double>9999999.000000000000000</double>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>exprBox</tabstop>
  <tabstop>authorBox</tabstop>
  <tabstop>reMinBox</tabstop>
  <tabstop>reMaxBox</tabstop>
  <tabstop>prMinBox</tabstop>
  <tabstop>prMaxBox</tabstop>
  <tabstop>fluidBox</tabstop>
  <tabstop>sectionBox</tabstop>
  <tabstop>angleBox</tabstop>
  <tabstop>borderBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddCorrelation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddCorrelation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
